WordPress/WordPressStatsWidgets/LockScreenWidgets/Views/LockScreenSingleStatView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+import WidgetKit
+
+@available(iOS 16.0, *)
+struct LockScreenSingleStatView: View {
+    @Environment(\.widgetFamily) var family: WidgetFamily
+    let viewModel: LockScreenSingleStatViewModel
+
+    var body: some View {
+        if family == .accessoryRectangular {
+            ZStack {
+                AccessoryWidgetBackground().cornerRadius(8)
+                VStack(alignment: .leading) {
+                    Text(viewModel.siteName)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .font(.system(size: 11))
+                        .lineLimit(1)
+                    Text(viewModel.value)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .font(.system(size: 20, weight: .bold))
+                        .foregroundColor(.white)
+                    Text("\(viewModel.title) \(viewModel.dateRange)")
```

Comment:
I wonder about different languages. Whether `title + dateRange` always makes sense, or there'll be languages where `dateRange + title` or any other combination makes sense.

Maybe we should just have a single String here which then can be appropriately localized? WDYT?

Reply:
Great point, I never think about this concern.
I prefer to treat it as "two" fields to make the feature simple for now because if we treat it as "one" field, then we need to provide many translations for the combination (field + date range)

And can probably improve the design later because one of the opinions from designers is there's another better design for them. Just pick this one as the safest design for the first iteration.
 
for example: 
| Large number | date range on top |
| - | - |
|<img width="123" alt="image" src="https://user-images.githubusercontent.com/3096210/225338653-975fbd02-094f-4bea-b5e7-1cd333c53e6d.png">|<img width="232" alt="image" src="https://user-images.githubusercontent.com/3096210/225339427-ccff8207-20e5-47c5-bb55-e343f8e6d4ac.png">|

So I suggest noting this concern as the discussion topic in the next phase, not blocking the current development progress, what do you think?

Reply:
> Great point, I never think about this concern.
I prefer to treat it as "two" fields to make the feature simple for now because if we treat it as "one" field, then we need to provide many translations for the combination (field + date range)

Got it, I understand where you're coming from. Since I12n is an important topic, I would like to take it into consideration. If we translate these labels separately and put one next to each other, the formulations may look incorrect in other languages. From the user's perspective, they won't see 2 different labels, they will see 2 words next to each other.

> And can probably improve the design later because one of the opinions from designers is there's another better design for them. Just pick this one as the safest design for the first iteration.

Yes, definitely we are always open to improvements 👍. However, since we chose a particular style of design for the moment, we should make sure it works well in different languages! 

> So I suggest noting this concern as the discussion topic in the next phase, not blocking the current development progress, what do you think?

I'm okay leaving it outside the scope of this PR but I prefer resolving the localization concern in this phase.

Reply:
No problem! I think it's good to create a specific key for this design style, and have the discussion in parallel, I will track the progress, so we can update the implementation depending on the new conclusion.